release-notes.md

```diff
@@ -175,15 +191,15 @@ The first version of Places is now available for public beta access!
 
 The Places solution is composed of the following components:
 
-* The **Places Services** is a robust set of REST APIs that provide an interface to the database that stores all Places-related data for a customer.  For more information, see [Places database management](https://launch.gitbook.io/places-services-by-adobe-documentation/places-database-management-1). 
-* A **Places-specific card** on Adobe IO \(coming soon\). 
+* The **Places Services** is a robust set of REST APIs that provide an interface to the database that stores all Places-related data for a customer.  For more information, see [Places database management](https://launch.gitbook.io/places-services-by-adobe-documentation/places-database-management-1).
```

Comment:
Not sure about the spacing here, but please make sure that "For more information..." is on the next line, under "The Places Services..."